modeltranslation/tests/tests.py

```diff
@@ -3606,3 +3606,47 @@ def test_managers_failure(self):
         ), "Permission is using MultilingualManager"
         user = User.objects.create(username='123', is_active=True)
         user.has_perm('test_perm')
+
+
+class SpecificLanguageModelTestCase(ModeltranslationTestBase):
+    def test_class_field(self):
+        models.SpecificLanguageModelX.objects.create(title_x_de="foo", text="bar")
+        models.SpecificLanguageModelX.objects.create(title_x_de="foo")
+
+        y_foo = models.SpecificLanguageModelX.objects.filter(title_x_de="foo")
+        assert 2 == y_foo.count()
+
+        y_baz = models.SpecificLanguageModelX.objects.filter(title_x_de="baz")
+        assert 0 == y_baz.count()
+
+    def test_register_decorator(self):
+        models.SpecificLanguageModelY.objects.create(title_y_de="foo", text="bar")
+        models.SpecificLanguageModelY.objects.create(title_y_de="foo")
+
+        y_foo = models.SpecificLanguageModelY.objects.filter(title_y_de="foo")
+        assert 2 == y_foo.count()
+
+        y_baz = models.SpecificLanguageModelY.objects.filter(title_y_de="baz")
+        assert 0 == y_baz.count()
```

Comment:
These two tests are essentially the same, different only model?

Replace it with single test and `@pytest.mark.parametrize("model", ...)`

Reply:
Btw, what this test supposed to check?

Reply:
Basically, this test checked whether an error occurs when creating a database record if a model with the languages parameter is registered.
It is expected that if something goes wrong, the model will not be able to write data or retrieve it. 
I will think about how to improve the tests to add more checks during the test.